src/daemon/libuv_workers.c

```diff
@@ -210,3 +210,74 @@ void release_cmd_pool(CmdPool *pool) {
     uv_mutex_destroy(&pool->lock);
     uv_cond_destroy(&pool->not_full);
 }
+
+/// Test
+
+typedef struct {
+    CmdPool *pool;
+    int total;
+    int failed;
+} ThreadArgs;
+
+void push_thread(void *arg) {
+    ThreadArgs *args = (ThreadArgs *)arg;
+    CmdPool *pool = args->pool;
+    for (int i = 0; i < args->total; ++i) {
+        cmd_data_t cmd;
+        snprintf(cmd.data, sizeof(cmd.data), "cmd-%d", i);
+        push_cmd(pool, &cmd, true);
+    }
+    fprintf(stderr, "PUSHED: %d commands\n", args->total);
+}
+
+void pop_thread(void *arg) {
+    ThreadArgs *args = (ThreadArgs *)arg;
+    CmdPool *pool = args->pool;
+
+    cmd_data_t cmd;
+    for (int i = 0; i < args->total; ) {
+        bool got = pop_cmd(pool, &cmd);
+        if (got) {
+            char expected[64];
+            snprintf(expected, sizeof(expected), "cmd-%d", i);
```

Comment:
As we’ve discussed, we don’t have `snprintfz` here, and we’re not reducing the second argument like we typically do in our code. Should we proceed with this new standard? :thinking:

Reply:
This is used only for the unittest function